CHANGELOG.md

```diff
@@ -1 +1,7 @@
+## [0.5.0] - 2020-03-27
 
+### Changed
+
```

Comment:
Extra line?

Reply:
Will https://github.com/aws-amplify/aws-sdk-ios/issues/2365 be same issue and can be solved by setting distribution flag?

Reply:
I don't think it matters because `-` will start a new block anyway, but I can remove this line to align with other CHANGELOGs.